packer/scylla_install_image

```diff
@@ -77,9 +77,13 @@ if __name__ == '__main__':
     else:
         print('no scylla package found.')
         sys.exit(1)
-
+    run('env', shell=True, check=True)
+    run('sleep 60', shell=True, check=True)
     run('apt-get update --allow-insecure-repositories -y', shell=True, check=True)
     run('apt-get full-upgrade -y', shell=True, check=True)
+    if args.target_cloud == 'aws':
```

Comment:
This specific bug we've seen is in AWS, but don't we want similar functionality (i.e., keep kernel 5.15) also for other clouds?

Reply:
GCP doesn't have lts for `22.04` - https://packages.ubuntu.com/jammy/linux-gcp

I see that Azure has, so I will implement it for both